molecule/rbac-mtls-rhel8/verify.yml

```diff
@@ -64,3 +65,15 @@
     - shell: "yum list available |grep Confluent.clients"
       register: client_package_grep
       failed_when: "client_package_grep.rc > 0"
+
+- name: Verify SSO Authentication
+  hosts: control_center
+  gather_facts: false
+  tasks:
+    - name: Check status of Authenticate api
+      uri:
+        url: "https://control-center1:9021/api/metadata/security/1.0/oidc/authenticate?caller=https://control-center1:9021/api/metadata"
+        validate_certs: false
+        follow_redirects: none
+        status_code: 302
+      register: sso
```

Comment:
as part of verification, we are testing that we get status code as 302 (which is redirect) - which means SSO is configured correctly and working? do we want to check anything explicitly in registered var - sso `register: sso` ?

also, do we have any negative tests? what happens when we misconfigure SSO details ? 🤔

Reply:
any result other than 302 would lead to failure of this task. And 302 return only comes when the redirection is successful